visualizations/frontend/other/TTable.vue

```diff
@@ -3,7 +3,7 @@
     <slot name="columnConfig" :setColumnConfig="setColumnConfig"></slot>
 
       <!-- Title -->
-      <div class="tableHeaderContainer">
+      <div class="tableHeaderContainer" v-if="title || tooltip  || enableCsvDownload || canSearch || (modifiableColumns && modifiableColumns.length > 0)">
```

Comment:
When I saw this I thought 'oh my, what will happen when we need to add another piece to this condition?'. 
We could move this to a computed property, however, we could also take a look at how we are using spacing on elements to create layout, and adjust things so that if an element isn't rendered it takes no space: 

I created an RFC to talk about this: https://github.com/topcoat-data/topcoat-public/pull/157

Reply:
Your RFC is fantastic, that is a MUCH better approach.